### DISCUSSION TITLE
How to Make Flexible Image Size?

### DISCUSSION THREAD

**Discussion by encient** [gethyas/hyas#450] [Q&A]
![image](https://github.com/gethyas/hyas/assets/95013757/1a6c0087-0361-405e-a523-58ae31dfa8a7)

My image is 856x137 pixels, and the template enlarged the image. May I know how to edit the settings to make the website flexible to image sizes?

**Comment by h-enk** [comment]
You could use the `width` attribute – see also https://images.gethyas.com/docs/built-ins/image-orientation/

**Reply by encient** [reply]
So Hyas by default does not allow image to be resized according to the webpage? Does that mean that I have to edit my images one by one to specify their size?

**Reply by h-enk** [reply]
You can set normalization width and height (using large images) in params.toml and use the width attribute for smaller images.

Hyas Images will be extended to support aspect ratios and more in the next release.

**Reply by encient** [reply]
Will updating it crash my previous blog? We can update by using this command `npm install @hyas/doks-core@latest` right

**Reply by h-enk** [reply]
Right and it will just work 😀

**Reply by encient** [reply]
Alright thanks

**Reply by encient** [reply]
May I know around when will it be released?

**Reply by h-enk** [reply]
Not sure yet, I will let you know next week

**Reply by encient** [reply]
Thanks, can't wait for the feature! I'm now waiting for this feature so I can upload all my contents into my website!
Cuz I still can't fix the image issue, this is my [repo](https://github.com/encient/encient.github.io)

**Reply by h-enk** [reply]
Confirmed — https://github.com/gethyas/images/issues/17

Planned fix to be available end of next week at the latest (depending on other priorities and availlable time)

**Reply by encient** [reply]
Thanks a lot, having lots of trouble with image now (all blurry)...
Will wait patiently for it haha XD

**Reply by encient** [reply]
Sorry for disturbing again, just to confirm this will also work for doks right? Cuz I'm using doks theme
